packages/wrangler/src/deploy/deploy.ts

```diff
@@ -386,20 +386,21 @@ export default async function deploy(props: Props): Promise<{
 			const serviceMetaData = await fetchResult<{
 				default_environment: {
 					environment: string;
+					script_tag: string;
 					script: {
 						tag: string;
 						tags: string[] | null;
 						last_deployed_from: "dash" | "wrangler" | "api";
-					};
+					} | null;
```

Comment:
What does it mean for script to be null?
In that case things like `last_deployed_from` are not defined so what does that mean? Should there be a default, like `api`?